src/main/scala/rocket/Instructions.scala

```diff
@@ -76,17 +76,26 @@ object Instructions {
   def C_J                = BitPat("b????????????????101???????????01")
   def C_JALR             = BitPat("b????????????????1001?????0000010")
   def C_JR               = BitPat("b????????????????1000?????0000010")
+  def C_LBU              = BitPat("b????????????????100000????????00")
```

Comment:
Actually, these are not used in IDecode. But it is OK to include them.

Reply:
Shouldn't this file be auto-generated?